floodviz/utils.py

```diff
@@ -0,0 +1,90 @@
+import requests
+
+"""
+    Takes a valid url and a dictionary of expected column header strings as keys (values do not matter).
+    parse_rdb then requests data from the service at url, and extracts data matching any header values.
+     
+    Data is returned as a list of dict objects, where each object represents a data point. 
+
+"""
+
+
+def parse_rdb(url, header_values):
```

Comment:
`header_values` should be passed in as a List if the values don't matter. You can always transform it into a dict of blank entries with `d = dict(zip(lst, ['' for _ in lst]))`, but here it's only being used as a list.